pkg/workflows/sdk/testutils/runner.go

```diff
@@ -79,7 +80,17 @@ func (r *Runner) Err() error {
 }
 
 func (r *Runner) ensureGraph(spec sdk.WorkflowSpec) error {
-	g, err := workflows.BuildDependencyGraph(spec)
+	proto, err := wasmpb.WorkflowSpecToProto(&spec)
```

Comment:
@ferglor Can you add a comment here explaining why you're doing this?

Eg `BuildDependencyGraph` expects non-aliased types as inputs in order to be able to generate the graph correctly. Serialize and deserialize the workflow to automatically convert the Spec to a supported format.